applications/CoSimulationApplication/custom_utilities/co_sim_io_conversion_utilities.cpp

```diff
@@ -92,6 +92,7 @@ const std::map<CoSimIO::ElementType, std::string> elem_name_map {
     {CoSimIO::ElementType::Pyramid3D13, "Element3D13N"},
     {CoSimIO::ElementType::Pyramid3D5, "Element3D5N"},
     {CoSimIO::ElementType::Quadrilateral2D4, "Element2D4N"},
+    {CoSimIO::ElementType::Quadrilateral3D4, "Element3D4N_Q"},
```

Comment:
Specifying the dimension and number of nodes will clearly no longer cut it so we'll have to settle on a new naming convention, and this time it better make more sense.

How about sticking the geometry's name in the front? For example
```
QuadElement3D4N
```

Reply:
@matekelemen I have changed the element name=) Let me know if it is good to be merged=)